source/_components/kira.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: page
+title: "Kira"
+description: "Instructions how to integrate Keene Electronics IR-IP modules (Kira) into Home Assistant."
+date: 2017-5-7 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: keene.png
+ha_category: Hub
+ha_release: 0.45
+---
+
+The `kira` component is the main component to integrate Keene Electronics IR over IP [Kira](https://www.keene.co.uk/keene-ir-anywhere-single-worldwide.html) modules with Home Assistant.
+
+An empty configuration will create a sensor using the default port that Kira modules use (UDP port 65432).
```

Comment:
Suggest some rewording since configuration is in general the whole of the config file and can cause some confusion for non native English speakers.
If anything I would suggest that you skip this line instead since the example configuration is commonly copy and pasted as a start for most new users.

You cover later after the code snippet why and how so that explains it rather well anyway.